cime_config/testdefs/testmods_dirs/clm/collapse_pfts_78_to_16_f10/include_user_mods

```diff
@@ -1 +0,0 @@
-../default
```

Comment:
This test mod doesn't have anything in it that is resolution dependent either.

Reply:
I'm keeping the suffix _f10 in the name of this test because this test points to a specific f10 surface dataset and would fail if we used a different resolution.

Reply:
To clarify, this is the test that I developed for PR #483 

It points to f10-specific surface and landuse datasets.

The only reason it appears in the current PR is that I added decStart to it, the way I did for the current PR's new test.

Reply:
Oh, sorry. I had to look at the full files to see that. Sorry for the wrong diversion.